src/components/receive/components/AddressBlock.tsx

```diff
@@ -171,6 +175,16 @@ const AddressBlock = ({
                   View on explorer <i className={["fas", "fa-external-link-square-alt"].join(" ")} />
                 </button>
               )}
+              {type === AddressType.transparent && (
```

Comment:
I think you need to check if there is something to shield, if the transparent balance is 0 make no sense to show this button.